writings/2021-12-12-the-importance-of-the-most-profitable-block.mdx

```diff
@@ -0,0 +1,86 @@
+---
+slug: on-the-most-profitable-block
+title: Why Building the Most Profitable Block is Important
+authors: [robert]
+tags: [flashbots]
+image: /img/transparency-january-1.png
+hide_table_of_contents: false
+---
+Central to the security of Ethereum is that every miner, and soon validator, can produce the most profitable block possible. Why is this? Why has Flashbots made it a goal to democratize access to the most profitable blocks? How are we doing so? This blog post answers these questions and more.
+<!--truncate-->
+
+## The changing role of miners and validators on Ethereum
+In the last year the job of miners on Ethereum underwent its largest shift since Ethereum’s inception. Previously miners optimized their blocks by ordering transactions they received by their gas price. There was little money to be made in investing in improving ordering by gas price. Instead miners focused on network-level optimizations, such as having a better view of the mempool or lowering the time it takes to receive new blocks.
+
+Today miners employ much more complicated ordering schemes since the widespread adoption of techniques for extracting MEV, or **maximal (formerly miner) extractable value**. MEV in this context is the value that can be captured by miners by ordering, inserting, or censoring transactions (for a formal definition of MEV see "[On the Formalization of MEV](https://writings.flashbots.net/research/formalization-mev/)"). For example, arbitraging the price difference between two assets on decentralized exchanges is a form of MEV. A miner maximizing their returns should optimize transactions in their blocks such that they capture as much MEV as possible, which juices their returns on top of the transaction fees they receive as well the block subsidy.
+
+Arbitrage is one type of MEV alongside liquidations, sandwiches, and a long tail of more bespoke MEV strategies. The total size of the MEV market annually is nearing a billion dollars on Ethereum alone today. Indeed, individual MEV opportunities can be worth many, many times more than the block reward or transaction fees. For example, one arbitrage bot landed 2 arbitrage transactions each worth over [$2m in a single day](https://twitter.com/bertcmiller/status/1448728068353368069) in October.
+
+To ensure Ethereum’s security it is essential that all miners can build blocks that capture these opportunities. If MEV extraction is not democratized then larger miners will be able to use their resources to optimize their MEV strategies more than other miners. With more advanced strategies larger miners will be able to generate more revenue from MEV than their peers. In turn these additional revenues can be used to grow larger, optimize more, and further cement their dominance in the network. As MEV extraction gets more complicated (e.g. cross-chain arbitrage) this dynamic is exacerbated because it will become more difficult to stay on the leading edge of MEV extraction.
+
```

Comment:
Perhaps not for here, but I feel we always make the implication "linear -> ok; superlinear -> catastrophic", where "(super)linear" is for the reward as a function of the input: MEV-less PoW/PoS are linear in the sense that for x% hashrate/validators, you get x% block rewards. MEV makes this superlinear. ok, but with linearity there is also a rich-get-richer dynamics at play, you could use your profits to buy more miners, increase your hashrate, and repeat--so is there an argument for why keeping it "linear" is crucial? or are we simply saying "it would be worse"?